src/HttpHandler/HttpHandlerFactory.php

```diff
@@ -24,24 +24,25 @@ class HttpHandlerFactory
     /**
      * Builds out a default http handler for the installed version of guzzle.
      *
-     * @param ClientInterface $client
+     * If a handler has been previously created, it will be returned instead
```

Comment:
This is outdated with the latest commits, could we please revert this as well as where we move `$version = ClientInterface::VERSION;`?